.github/ISSUE_TEMPLATE/report-an-issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Report an issue
+about: Help us improve our service by reporting issues you are experiencing.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what is the issue.
```

Comment:
Recommend making these comments so that they don't show up if the user leaves it and submits the issue without removing these descriptions:

```
**Describe the issue**
<!-- A clear and concise description of what is the issue. -->
```